pkg/karmadactl/label/label.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2024 The Karmada Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package label
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	kubectllabel "k8s.io/kubectl/pkg/cmd/label"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	"github.com/karmada-io/karmada/pkg/karmadactl/util"
+)
+
+var (
+	labelExample = templates.Examples(`
+		# Update propagationpolicy 'foo' with the label 'priority' and the value '0'
+		[1]%s label pp foo priority=0
```

Comment:
I'd say this seems not a good example, as this label might be misleading with the priority in PropagationPolicy.

Actually, we have some real-world labels users might use, like `resourcetemplate.karmada.io/deletion-protected`.